frontend/Dockerfile

```
@@ -0,0 +1,60 @@
+
+# =============================================================================
+# STAGE 2: DEVELOPMENT RUNTIME
+#
+# Start fresh with minimal Node.js image and only copy what's needed
+# to run the Vite dev server.
+# =============================================================================
+
+FROM node:${NODE_BASE}
+
+# Install curl for healthcheck
+RUN apk add --no-cache curl
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)